boot/src/test/java/com/taobao/arthas/boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,39 @@
+package com.taobao.arthas.boot;
+
+import com.taobao.arthas.common.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+
+import static com.taobao.arthas.boot.DownloadUtils.*;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testReadMavenReleaseVersion() {
+        Assert.assertNull(readMavenReleaseVersion(""));
+    }
+
+    @Test
+    public void testReadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), readAllMavenVersion(""));
+    }
+
+    @Test
+    public void testGetRepoUrl() {
+        Assert.assertEquals("http", getRepoUrl("https/", true));
+        Assert.assertEquals("https://repo1.maven.org/maven2", getRepoUrl("center", false));
+        Assert.assertEquals("https://maven.aliyun.com/repository/public", getRepoUrl("aliyun", false));
+    }
+
+    @Test
+    public void testReadMavenMetaData() throws IOException {
+        Assert.assertEquals(IOUtils.toString(new URL("http://repo1.maven.org/maven2/com/taobao/arthas/arthas-packaging/maven-metadata.xml").openStream()), readMavenMetaData("center", true));
```

Comment:
Break lines > 80 chars

Reply:
Hi Peter,
Would you like me to create a string variable for the url to shorten the assert?

Reply:
for example